### PR TITLE
Add auto tests, lower swift tool version to 5.10.0, update README

### PR DESCRIPTION
**Pull request by ningkaiqiang**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+A Swift implementation of FSRS5.0
+
+[![codecov](https://codecov.io/gh/ningkaiqiang/swift-fsrs/graph/badge.svg?token=DQSJTROPFG)](https://codecov.io/gh/ningkaiqiang/swift-fsrs)
```
Comment (ishiko732) [inline README.md:3]:
```suggestion
[![codecov](https://codecov.io/gh/open-spaced-repetition/swift-fsrs/graph/badge.svg?token=K2C0Z5PFEH)](https://codecov.io/gh/open-spaced-repetition/swift-fsrs)
```

Comment (ningkaiqiang) [reply README.md:3]: I think this is more reasonable, please update the github actions secrets config.

Comment (L-M-Sherlock) [reply README.md:3]: @ishiko732, what's the token of our org?

Comment (ishiko732) [reply README.md:3]: > @ishiko732, what's the token of our org?

we can view this link: https://app.codecov.io/gh/open-spaced-repetition/swift-fsrs/config/badge

**Pull request merged**